src/unzck.c

```diff
@@ -58,7 +58,7 @@ struct arguments {
   char *args[1];
   zck_log_type log_level;
   bool dict;
-  bool stdout;
+  bool stdOut;
```

Comment:
Is there a reason you're renaming this variable?  My preference is to join variables using underscores rather than camelCase, but, in this case, I think I'd rather it just stay stdout.

Reply:
Yes indeed. In fact on musl libc "stdout" is a preprocessor macro whose expansion leads to
compilation errors:

```
| In file included from ../git/src/unzck.c:31:
| ../git/src/unzck.c: In function 'parse_opt':
| ../git/src/unzck.c:78:24: error: expected identifier before '(' token
|    78 |             arguments->stdout = true;
|       |                        ^~~~~~
| ../git/src/unzck.c: In function 'main':
| ../git/src/unzck.c:141:20: error: expected identifier before '(' token
|   141 |     if(!(arguments.stdout)) {
|       |  
```

So, as already done by libvirt with stderr (https://github.com/ligurio/libvirt/commit/44d63ad997216f05f5c7ce005acc308bbdd00458), I renamed stdout to stdOut, good for you ?

Reply:
Ok, that makes sense.  I realize it's a nitpick, but the rest of the code uses `snake_case`, so could you switch it to `std_out`, please?

Reply:
Sure no problem ;), PR updated, thanks for the review @jdieter